src/main/java/org/jabref/gui/openoffice/ManualConnectDialog.fxml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ButtonType?>
+<?import javafx.scene.control.DialogPane?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.RowConstraints?>
+
+
+<DialogPane prefHeight="130.0" prefWidth="317.0" xmlns="http://javafx.com/javafx/8.0.141" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.jabref.gui.openoffice.ManualConnectDialogView">
+   <content>
+      <GridPane minWidth="-Infinity" prefHeight="50.0" prefWidth="585.0">
```

Comment:
Please try to not use a GridPane for layouts and consider puting the labels above the text fields (https://ux.stackexchange.com/questions/8480/input-form-label-alignment-top-or-left)

Reply:
This dialog is only shown rarely. Therefore, according to your link a grid pane is okay.
And the content differs depending on the OS (e.g. select one folder vs multiple files)